charts/fulcio/values.schema.json

```diff
@@ -445,6 +445,14 @@
                         }
                     ]
                 },
+                "livenessProbe": {
+                    "description": "Liveness probe configuration",
+                    "$ref": "https://kubernetesjsonschema.dev/v1.18.1/_definitions.json#/definitions/io.k8s.api.core.v1.Probe"
+                },
+                "readinessProbe": {
+                    "description": "Readiness probe configuration",
+                    "$ref": "https://kubernetesjsonschema.dev/v1.18.1/_definitions.json#/definitions/io.k8s.api.core.v1.Probe"
```

Comment:
Any thoughts on using these references to https://kubernetesjsonschema.dev? Not sure what implications this has, as I don't see any `$ref` references within the Sigstore Helm charts.

Reply:
How did you generate these values ?